task-2/src/components/SearchBar/uselocationTest.js

```diff
@@ -0,0 +1,17 @@
+// import React from 'react';
```

Comment:
Consider NOT leaving redundant commented code in your app.

Reply:
Resolved.